pages/admin/event/[id].vue

```diff
@@ -0,0 +1,24 @@
+<script setup lang="ts">
+import 'tailwindcss/src/css/preflight.css'
```

Comment:
Hmmm is preflight not imported from default Tailwind Nuxt plugin?

Reply:
It works when navigating from say /admin to /event/[:id], but if the user goes to /event/[:id] directly, it doesn't seem to be imported